iceoryx2/tests/service_publish_subscribe_tests.rs

```diff
@@ -864,7 +864,7 @@ mod service_publish_subscribe {
         const BUFFER_SIZE: usize = 1;
         const HISTORY_SIZE: usize = 0;
         const MAX_BORROW: usize = 1;
-        const MAX_SUBSCRIBERS: usize = 100;
+        const MAX_SUBSCRIBERS: usize = 50;
```

Comment:
I'm thinking whether it would make sense to keep the old value on Linux as canary check that something is broken?

Reply:
At this level I would like to have the same tests, behavior, limits etc. and say with confidence that every platform supports at least 50 subscribers.

I completely understand your rational, but when we go this route we may have a wild mixture of verified upper limits for every platform. I would rather go the path that we say 50 subscribers are supported for every platform, but an individual higher limit for a specific platform is maybe working.

Maybe we can introduce platform specific system limit tests where exactly those differences can be verified. What do you think?

Reply:
Indeed. These limits might be a criteria for tier 2 and for tier 1 we could have tests for additional limits.